src/module.c

```diff
@@ -2434,7 +2434,7 @@ void VM_Yield(ValkeyModuleCtx *ctx, int flags, const char *busy_reply) {
                  * loop (ae.c) and avoid potential race conditions. */
 
                 int acquiring;
-                atomicGet(server.module_gil_acquring, acquiring);
+                atomicGet(server.module_gil_acquiring, acquiring);
```

Comment:
It looks like you forgot to replace this `atomicGet` with `atomic_load_explicit` here.